src/main/java/frc/robot/subsystems/ElevatorArmSubsystem.java

```diff
@@ -44,16 +45,19 @@ public enum Setpoint {
     kL4;
   }
 
+  // Elevator Motor
   private SparkFlex elevatorMotor =
       new SparkFlex(ElevatorConstants.ELEVATOR_MOTOR, MotorType.kBrushless);
   private SparkClosedLoopController elevatorClosedLoopController =
       elevatorMotor.getClosedLoopController();
+  private RelativeEncoder elevatorEncoder = elevatorMotor.getExternalEncoder();
 
+  // Arm Motor
   private SparkFlex armMotor = new SparkFlex(ArmConstants.ARM_MOTOR, MotorType.kBrushless);
-  private RelativeEncoder elevatorEncoder = elevatorMotor.getEncoder();
+  private RelativeEncoder armEncoder = armMotor.getExternalEncoder();
```

Comment:
```suggestion
  private SparkAbsoluteEncoder armEncoder = armMotor.getAbsoluteEncoder();
```

Maybe do it here too?